base/cvd/cuttlefish/host/commands/cvdalloc/cvdalloc.cpp

```diff
@@ -131,6 +139,9 @@ Result<int> CvdallocMain(int argc, char *argv[]) {
   CF_EXPECT(Wait(sock, kSemNoTimeout));
 
   /* Teardown invoked by scopeguard above. */
+  /* This isn't a dead store. */
```

Comment:
The anonymous function used with `ScopeGuard` above is taking `clean_teardown` by value rather than by reference. To make sure modifications are propagated, it needs to be `&clean_teardown` in the capture list. Does that fix the lint error as well?

Reply:
Ah, of course.